assets/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "devDependencies": {
+    "html5-qrcode": "^2.3.7",
```

Comment:
`devDependencies` is the list of packages only needed for local development and testing. So that, this should be added to the `dependencies` list. Learn more [here](https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file).

Reply:
I was messing around with npm just now and found out apparently the other dependencies were simply copied to vendor/ instead of managed using npm (https://github.com/joinum/parzival/commit/28a057923f652137d41b3a96baf408bbf0c181b2). Should I do the same here? It doesn't seem like a great practice and I can't find why it was originally done.